_docs/1.0.0/usage-schema.md

```diff
@@ -0,0 +1,618 @@
+| | |
+
+Time zone of the timestamp or date (for later it's generally not recommended). For details see the chapter
+[Parsing timestamps and dates](#parsing-timestamps-and-dates).
+
+In case the [`pattern`](#pattern) already includes information to recognize the time zone, the `timezone` entry in _metadata_ will
+be ignored. Namely if the pattern includes *"z"*, *"Z"* or *"X"* placeholder or *"epoch"*, *"epochmilli"*, *"epochmicro"*
+ and *"epochnano"* keywords.
+
+**NB!** Due to spark limitation, only time zone ids are accepted as valid values. To get the full list of supported time
+ zone denominators see the output of Java's
+[`TimeZone.getAvailableIDs()` function](https://docs.oracle.com/javase/8/docs/api/java/util/TimeZone.html#getAvailableIDs--).
+
+### decimal_separator
+
+| <sub><sup>Supported by types:</sup></sub> | <sub><sup>[Decimal](#decimal), [Long](#long), [Integer](#integer), [Short](#short), [Byte](#byte), [Double](#double), [Float](#float)</sup></sub> |
+|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| | |
+
+The character separating the integer and the fractional parts of the number.
+
+Seems pointless for whole numbers, but in case *"."* is used for example as
+[`grouping_separator`](#grouping_separator) the , the `decimal_separator` has to
+be redefined, to avoid conflict.
+
+### grouping_separator
+
+| <sub><sup>Supported by types:</sup></sub> | <sub><sup>[Decimal](#decimal), [Long](#long), [Integer](#integer), [Short](#short), [Byte](#byte), [Double](#double), [Float](#float)</sup></sub> |
+|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| | |
+
+Character to mark boundaries between orders of magnitude, usually to mark
+thousands, millions etc.
+
+It has to be used in the pattern to be taken into consideration.
+
+### minus_sign
+
+| <sub><sup>Supported by types:</sup></sub> | <sub><sup>[Decimal](#decimal), [Long](#long), [Integer](#integer), [Short](#short), [Byte](#byte), [Double](#double), [Float](#float)</sup></sub> |
+|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| | |
+
+Character to mark the number is negative. By default, it’s the standard minus
+sign (*"-"*).
+
+### allow_infinity
+
+| <sub><sup>Supported by types:</sup></sub> | <sub><sup>[Double](#double), [Float](#float)</sup></sub> |
+|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| | |
+
+Flag indicating if the column accepts infinity as a value. Also too big/far
```

Comment:
```suggestion
Flag indicating if the column accepts infinity as a value. When set to true, positive and negative numbers with values that are too large are converted to *infinity* and *-infinity*, respectively.
```

Reply:
Reworded to:
```
Flag indicating if the column accepts infinity as a value. When set to true, positive and negative numbers with values
that are too large are converted to *infinity* and *-infinity*, respectively. Also the string for infinity _"∞"_ is 
recognized and accepted.
```

Reply:
Looks good, but you should try removing every `also` you write. I promise you almost no sentence will miss it.